### ISSUE TITLE
Revoked access to the app, from Gmail

### ISSUE THREAD

**Issue by karipidist** [jay0lee/got-your-back#195]
**I just got this from Gmail:**

"Hello,

Although you do not need to take any action, we'd like to let you know that the third-party apps below will no longer have access to some of your Google Account data, including your Gmail content. This change will take effect from March 31, 2019.
Got Your Back: Gmail Backup

We are making this change as part of our ongoing efforts to ensure that your data is protected and private. These applications have not yet complied with the updated data privacy requirements announced on October 8, 2018"

**Comment by MagicAndi** [comment]
Can confirm - I also received the same email this morning.

**Comment by dazbradbury** [comment]
Does anyone know what changes / have a summary of how to make GYB compliant?

**Comment by dbirchbauer** [comment]
See https://developers.googleblog.com/2018/10/elevating-user-trust-in-our-api.html specifically the Application Review section I think its just some paperwork on google's end

**Comment by jay0lee** [comment]
GYB 1.20 is now available. Basic instructions for installing and fixing the issue are in the release notes:

https://github.com/jay0lee/got-your-back/releases

can a few people try upgrading and fixing and give me feedback on the process?

**Comment by mneiger** [comment]
here is what I get at the last step gyb --action quota --email 

```
Traceback (most recent call last):
  File "gyb.py", line 2026, in <module>
  File "gyb.py", line 1365, in main
  File "gyb.py", line 321, in requestOAuthAccess
  File "oauth2client/_helpers.py", line 133, in positional_wrapper
  File "oauth2client/client.py", line 2135, in flow_from_clientsecrets
  File "oauth2client/clientsecrets.py", line 165, in loadfile
  File "oauth2client/clientsecrets.py", line 122, in _loadfile
  File "json/__init__.py", line 296, in load
  File "json/__init__.py", line 348, in loads
  File "json/decoder.py", line 337, in decode
  File "json/decoder.py", line 353, in raw_decode
json.decoder.JSONDecodeError: Expecting ',' delimiter: line 11 column 1 (char 467)
[2471] Failed to execute script gyb


```

**Comment by mneiger** [comment]
my bad, and extra "`" somehow ended up in the client_secrets.json file. 
It works now.

**Comment by jay0lee** [comment]
Yes, that's fixed in 1.21.

**Comment by genem2** [comment]
> GYB 1.20

Appears fine here, and awesome job with the instructions!

I use GYB to backup a folder from my primary account, and restore it to a secondary GMail account. (I do not have G Suite.)
I appears I'll need to do more than simply running gyb-setup.bat for the second email address.

Should I just setup GYB in seperate folders for that, and set up two projects?
And, maybe when that's done, is there a way to merge the json files to use both in the same directory?

Thanks,
Appears you Do _have our backs_!
